src/assets/custom/css/_sass/_website-navigation-sub-menus.scss

```diff
@@ -0,0 +1,121 @@
+.website-navigation-sub-menus__menu {
+  @include shared-navigation-menu-spacing;
+  list-style-type: none;
+  margin: 0;
+  position: absolute;
+
+  @include small-and-medium {
+    left: 100vw;
```

Comment:
Can we double check this isn't causing some overflow?

Reply:
Good call. I've addressed that issue here: https://github.com/codurance/site/pull/1921/commits/8edc9ffe510bf156a921a0102463280067716e63